selenium/src/test/java/au/org/emii/portal/tests/landing/AcknowledgementTest.java

```diff
@@ -0,0 +1,17 @@
+package au.org.emii.portal.tests.landing;
+
+import au.org.emii.portal.tests.BaseTest;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+
+public class AcknowledgementTest extends BaseTest {
+
+    @Test
+    public void linkTest() {
+        webElementUtil.clickLinkContainingText("Acknowledgement");
```

Comment:
Is there any way to link these text comparisons with the values in en.js within the main portal? Otherwise, as soon as the wording changes we're going to break a stack of tests. It would be best not to duplicate simple text config if at all possible.